params/config.go

```diff
@@ -37,9 +37,11 @@ var (
 var (
 	XDPoSV2Config = &V2{
 		TimeoutWorkerDuration: 50000,
+		CertThreshold: 73, // must be > common.MaxMasternodesV2 * 2 / 3
```

Comment:
just give the right formula, something like below, otherwise we may forget to change this number if common.MaxMasternodesV2 gets updated.
```
CertThreshold:  common.MaxMasternodesV2 * 2 / 3 + 1 
```